tests/test_config.py

```diff
@@ -319,11 +326,11 @@ def test_get_target_conf_custom_defaults(self):
         """)
         self.assertEqual(
             self.conf.get_target_conf('shproj'),
-            _target(123, apikey='key', stack='custom_default')
+            _target(123, apikey='key', stack='custom_default', eggs=[])
```

Comment:
Yeah, we should at least check if it reads/sends the data from `egg1.egg` and `egg2.egg` properly in `test_deploy`. That's also missing for the `requirements_file` parameter. If you want and have time you can add them to this PR @krotkiewicz, otherwise I can make a follow-up PR.